Program_Licenses.md

```diff
@@ -37,6 +37,7 @@ The licenses of the open-source software that is contained in these Docker image
 | Mash | non-standard license (see link) | https://github.com/marbl/Mash/blob/master/LICENSE.txt |
 | mashtree | GNU GPLv3 | https://github.com/lskatz/mashtree/blob/master/LICENSE |
 | Medaka | Mozilla Public License 2.0 | https://github.com/nanoporetech/medaka/blob/master/LICENSE.md |
+| Metaphlan | non-standard license (see link) | https://github.com/biobakery/MetaPhlAn/blob/3.0/license.txt |
```

Comment:
I believe they do use the standard MIT license

Reply:
you're right, thanks for catching that. I'll fix it